bigchaindb/core.py

```diff
@@ -45,17 +40,11 @@ def __init__(self, public_key=None, private_key=None, keyring=[], connection=Non
         Args:
             public_key (str): the base58 encoded public key for the ED25519 curve.
```

Comment:
Makes sense to also remove the descriptions of `public_key` and `private_key`.

Reply:
done.